### PR TITLE
Template, Refactor code

### PR DESCRIPTION
**Pull request by Mazende1990**

### CODE REVIEW

File: src/blocks/ale/Bortforslingdodsboale.js [hunk 1]
```diff
@@ -1,128 +1,26 @@
...
+const Bortforslingdodsboale = () => {
+
+  const serviceData = ({
+    mainquery: "bortforsling dödsbo Ale",
     metaTitle: "Smidig bortforsling av dödsbo i Ale - Låt oss ta hand om allt",
-    metaContent:
-      "Behöver du hjälp med bortforslingen av dödsbo i Ale. Kontakta oss idag för en pålitlig bortforsling av dödsbo i Ale.",
+    metaContent: "Behöver du hjälp med bortforslingen av dödsbo i Ale. Kontakta oss idag för en pålitlig bortforsling av dödsbo i Ale.",
     query: "Bortforsling dödsbo Ale",
+    city: "Ale",
```
Comment (Abbe91) [inline src/blocks/ale/Bortforslingdodsboale.js:12]: Can you explain this line why we have it now and we don't before?

File: src/blocks/ale/Hjalpdodsboale.js [hunk 2]
```diff
@@ -1,127 +1,23 @@
...
+
+
+const Hjalpdodsboale = () => { 
+
+  const serviceData = ({
+    mainquery: "Hjälp dödsbo Ale",
     metaTitle: "Hjälp dödsbo Ale. Snabb & smidig hjälp med dödsbon.",
     metaContent: "Hjälp dödsbo Ale. Snabb & smidig hjälp med dödsbon.",
     query: "Hjälp dödsbo Ale",
+    city: "Ale",
```
Comment (Abbe91) [inline src/blocks/ale/Hjalpdodsboale.js:12]: the same here

**Review by Abbe91 — COMMENTED** [review]
Please answer the comment

File: src/blocks/ale/Bortforslingdodsboale.js (hunk 1 above)
Comment (Mazende1990) [reply src/blocks/ale/Bortforslingdodsboale.js:12]: Explained in meeting

File: src/blocks/ale/Hjalpdodsboale.js (hunk 2 above)
Comment (Mazende1990) [reply src/blocks/ale/Hjalpdodsboale.js:12]: Explained in meeting

**Review by Abbe91 — APPROVED** [review]

**Pull request merged**